src/main/scala/LilaHandler.scala

```diff
@@ -50,9 +50,9 @@ final class LilaHandler(
       )
     case ApiUserOnline(user, false) => users.tellOne(user, ClientCtrl.ApiDisconnect)
 
-    case Impersonate(user, by) => Impersonations(user, by)
-
-    case Pong(pingAt) => Monitor.ping.record("site", pingAt)
+    case Impersonate(user, by)                                   => Impersonations(user, by)
+    case StreamersOnline(streamers: Iterable[(User.Id, String)]) => Streamer.set(streamers)
```

Comment:
I don't think there's a need or benefit to repeating the `streamers` type here

Reply:
I agree, let's get rid of some of these types!